api/pkg/apis/v1alpha1/providers/stage/materialize/materialize.go

```diff
@@ -187,32 +187,6 @@ func (i *MaterializeStageProvider) Process(ctx context.Context, mgrContext conte
 						return outputs, false, v1alpha2.NewCOAError(nil, fmt.Sprintf("Empty instance name: catalog - %s", name), v1alpha2.BadRequest)
```

Comment:
You can still print error like failed to create xxx. Can we assume the error is "empty instance name?"